JoinColony/colonyCDapp
# feat: added createdAt field

## Description

- Added created at row to history cards in payment builder flow

![image](https://github.com/JoinColony/colonyCDapp/assets/76940796/af2bc6aa-b063-485d-98b2-659ff8988dee)

## Testing

* Create a Payment builder action, using Permissions for each step.
* Click back through each step and check the history cards.

Resolves [2413](https://github.com/JoinColony/colonyCDapp/issues/2413)
